packages/@webex/plugin-meetings/src/locus-info/parser.ts

```diff
@@ -690,7 +690,7 @@ export default class Parser {
    */
   private stopSyncTimer() {
     if (this.syncTimer !== null) {
-      clearTimeout(this.syncTimer);
+      clearTimeout(this.syncTimer as number);
```

Comment:
this doesn't seem right, wouldn't it be better to fix the type declaration for syncTimer? Like this:
`syncTimer?: ReturnType<typeof setTimeout>`
(and also change all the places that use null with syncTimer to undefined)

Reply:
This is done @marcin-bazyl